homu/server.py

```diff
@@ -36,10 +36,38 @@ def find_state(sha):
     raise ValueError('Invalid SHA')
 
 
+def remove_state(state):
+    """Remove state"""
+    pull_num = state.num
+    repo_label = state.repo_label
+
+    del g.states[repo_label][pull_num]
+
+    db_query(g.db, 'DELETE FROM pull WHERE repo = ? AND num = ?',
+             [repo_label, pull_num])
+    db_query(g.db, 'DELETE FROM build_res WHERE repo = ? AND num = ?',
+             [repo_label, pull_num])
+    db_query(g.db, 'DELETE FROM mergeable WHERE repo = ? AND num = ?',
+             [repo_label, pull_num])
+
+
+def require_listen_deployment(repo_cfg):
+    return repo_cfg.get("buildbot", {}).get("deployment")
+
+
+def find_deployment(state, sha):
+    for deployment in state.get_repo().iter_deployments(10):
```

Comment:
Please add a comment on why the magic number is 10 or else I will feel confused every time I look at this line of code :)